goodtablesio/plugins/github/tasks.py

```diff
@@ -2,56 +2,32 @@
 import os
 import subprocess
 import tempfile
-import uuid
 import logging
-from flask import Blueprint, request, abort
-from .. import exceptions
-from .. import helpers
-log = logging.getLogger(__name__)
-
-
-SECRET = 'HUGI#6A2X|e{.Rkn`zg?a!`/9(&(Y7WYQqW#5.(&][s&-W(A8y;85pC:&;H<v*aw'
-CLONE_DIR = '/tmp'
-
 
-github = Blueprint('github', __name__, url_prefix='/github')
+from goodtablesio import helpers
+from goodtablesio.tasks import app as celery_app
 
 
-@github.route('/hook', methods=['POST'])
-def create_task():
+log = logging.getLogger(__name__)
 
-    # TODO: check origin with secret
-    # TODO: process errors
+TABULAR_EXTENSIONS = ['csv', 'xls', 'xlsx', 'ods']
+CLONE_DIR = '/tmp'
 
-    # Get payload
-    payload = request.get_json()
-    if not payload:
-        abort(400)
 
-    # Get task id
-    task_id = str(uuid.uuid4())
+@celery_app.task(name='goodtablesio.github.clone_repo_files')
+def clone_repo_files(clone_url, task_id):
```

Comment:
In my opinion it's more like `get_task_desc` or something like this. Because it returns task descriptor.